src/ray/object_manager/object_manager.cc

```diff
@@ -300,36 +300,9 @@ void ObjectManager::HandleSendFinished(const ObjectID &object_id, const NodeID &
                  << ", status: " << status.ToString();
   if (!status.ok()) {
     // TODO(rkn): What do we want to do if the send failed?
+    RAY_LOG(ERROR) << "Failed to send a push request for an object " << object_id
```

Comment:
This will be too spammy--- please remove or set to DEBUG.

Reply:
Ah yeah good point... I blindly made it error for some reasons... Let me fix it.